resin/resin/core/window.cpp

```diff
@@ -128,7 +128,8 @@ void Window::set_glfw_callbacks() const {
   glfwSetMouseButtonCallback(window_ptr_, [](GLFWwindow* window, int button_code, int action, int /*mods*/) {
     const WindowProperties& properties = *static_cast<WindowProperties*>(glfwGetWindowUserPointer(window));
 
-    mouse::Code button = mouse::kMouseCodeGLFWMapping.from_value(button_code);
+    // this value is always present in this context, as all GLFW buttons are properly mapped
+    mouse::Code button = mouse::kMouseCodeGLFWMapping.from_value(button_code).value();
```

Comment:
I've read the comment, but still I would do the following, it really cheap

```suggestion
    auto button = mouse::kMouseCodeGLFWMapping.from_value(button_code);
    if (!button) {
      return;
   }
```

just in case this mf will get glfw support: 

![image](https://github.com/user-attachments/assets/eeb9ecee-f44b-42d9-9024-beeeb6ae4ee6)

Reply:
Okey-dokey